src/pages/purchase-orders/edit/components/Actions.tsx

```diff
@@ -217,13 +221,19 @@ export function Actions(props: Props) {
       {actions().map(
         (action, index) =>
           !action(props.purchaseOrder).hideIf && (
-            <DropdownElement
-              key={index}
-              onClick={action(props.purchaseOrder).onClick}
-              icon={action(props.purchaseOrder).icon}
-            >
-              {action(props.purchaseOrder).label}
-            </DropdownElement>
+            <>
+              {action(props.purchaseOrder).showDivider && (
+                <Divider withoutPadding />
+              )}
+
+              <DropdownElement
+                key={index}
+                onClick={action(props.purchaseOrder).onClick}
+                icon={action(props.purchaseOrder).icon}
+              >
+                {action(props.purchaseOrder).label}
+              </DropdownElement>
+            </>
```

Comment:
Can we refactor this using std `ResourceActions`?

https://github.com/invoiceninja/ui/blob/main/src/components/ResourceActions.tsx

Reply:
Sure, the code has been refactored with the current convenction of handling more_actions.